clubs_guides/forms.py

```diff
@@ -0,0 +1,49 @@
+from django import forms
+from django.forms import widgets
+
+from clubs_guides.models import ClubsGuide
+
+
+class ClubsGuideForm(forms.ModelForm):
+    is_translation = forms.ChoiceField(
+        label="Is this guide a translation of another guide?",
+        widget=widgets.RadioSelect,
+        choices=(
+            (True, "Yes"),
+            (False, "No"),
+        ),
+        initial=False,
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(ClubsGuideForm, self).__init__(*args, **kwargs)
+
+        if "instance" in kwargs and kwargs["instance"] is not None:
+            self.fields["is_translation"].initial = (
+                kwargs["instance"].translation_of is not None
+            )
+
+    def clean_category(self):
+        cleaned_data = self.cleaned_data
+        category = cleaned_data.get("category")
+        if cleaned_data.get("is_translation") == "True":
+            category = self.clean_translation_of().category
```

Comment:
just thinking in worse-cases, what happens if clean_translation_of returns `None`? is that a crash with a server 500? (say someone submits the form without having picked an actual translation, just hits "save" immediately)